website/docs/r/qumulo_file_system.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+subcategory: "Qumulo"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_qumulo_file_system"
+description: |-
+  Manages a File System.
+---
+
+# azurerm_qumulo_file_system
+
+Manages a Qumulo File System.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.0/24"]
+
+  delegation {
+    name = "delegation"
+    service_delegation {
+      actions = ["Microsoft.Network/virtualNetworks/subnets/join/action"]
+      name    = "Qumulo.Storage/fileSystems"
+    }
+  }
+}
+
+resource "azurerm_qumulo_file_system" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  admin_password      = ")^X#ZX#JRyIY}t9"
```

Comment:
Can we change the example value here to something that's _very_ obviously a placeholder? e.g.

```suggestion
  admin_password      = "$uPeRSeCr3tCh@nGeMe"
```